### ISSUE TITLE
Mansaka recipe

### ISSUE THREAD

**Issue by githubteacher** [githubteacher/octo-recipes#3]
Hey all,

I though we should improve our Danish presence in the website so can we create a new Mansaka recipe?

![429553](https://cloud.githubusercontent.com/assets/2132216/9719470/70d7cdf4-557c-11e5-8927-3af41d43eea4.jpg)

Cheers,

:octocat: :heart:

**Comment by bitoiu** [comment]
I will give it a go, I will try to create this one! :smile:

**Comment by bitoiu** [comment]
@bnferguson can you take a look at this one?

**Comment by bitoiu** [comment]
@githubteacher did you see my previous comment?

**Comment by bnferguson** [comment]
I'm not super familiar with Mansaka but I'll see what I can do!

**Comment by bnferguson** [comment]
Should I take https://github.com/githubteacher/octo-recipes/issues/4 into account and also add calorie info?

**Comment by PhillipDenmark** [comment]
delicious

**Comment by kkh9000** [comment]
Not a good idea. Calorie info is depressing.

**Comment by kkh9000** [comment]
Not a good idea. Calorie info is depressing.

**Comment by bitoiu** [comment]
> Not a good idea. Calorie info is depressing.

:smile: that's true.

**Comment by drvenabili** [comment]
@bnferguson Is that a really big Pringle in your profile picture?

**Comment by Bunnymand** [comment]
:cry: : I am on a diet

**Comment by bnferguson** [comment]
> Not a good idea. Calorie info is depressing.

Good point. I'll leave it out for now. :no_good: :cake: :chart_with_upwards_trend:

**Comment by audedieude** [comment]
:+1:

**Comment by elektrobohemian** [comment]
:scream_cat:

**Comment by bnferguson** [comment]
@bnferguson - Ha! Common question. It is not, it's gouda!

**Comment by bnferguson** [comment]
@faustusdotbe - Ha! Common question. It is not, it's gouda!

**Comment by MadeHube** [comment]
@vcruzdelvalle Roger that ?

**Comment by vcruzdelvalle** [comment]
@MadeHube Got it (highfive)

**Comment by karenhyt** [comment]
@danielpryn - mon det er en vegetarret?

**Comment by danielpryn** [comment]
@karenhyt rimer på dyt :laughing:

**Comment by SenstH** [comment]
I prefer Currywurst

**Comment by karenhyt** [comment]
@Bunnymand du bliver foreslået

**Comment by melbaek** [comment]
This would probably go well with :beers:

**Comment by karenhyt** [comment]
and a :hamburger:

**Comment by mosart** [comment]
@aledl you should stop be ing so funny about your wife! ;)

**Comment by danielpryn** [comment]
### Look at me, @karenhyt
I'm making a list:

* First item
* Second item

:facepunch:

**Comment by Bunnymand** [comment]
@karenhyt Det gør du også vi må være vigtige :tophat:

**Comment by szurcher15** [comment]
@ingerpinger please add your comment on my comment not saying anything?

**Comment by ALedl** [comment]
@mosart I'm not married at all.

**Comment by danielpryn** [comment]
Is that you, @karenhyt ?

![GitHub Logo](http://orig05.deviantart.net/12ff/f/2012/201/3/7/pixel_lady_gaga_by_fightingferret-d580wis.gif)

**Comment by kkh9000** [comment]
> This would probably go well with :beers:

@melbaek It sure will :beers: :beers: :beers: :beers: :beers: :beers:

**Comment by karenhyt** [comment]
@danielpryn it is me :dancer:

**Comment by drvenabili** [comment]
- [x] Get a GitHub account
- [x] Comment on issues
- [ ] Go drink some :beers:

**Comment by mosart** [comment]
:smile:

**Comment by JeannetteE** [comment]
this is not magic or rocket science, right:)

**Comment by kkh9000** [comment]
:rocket: @JeannetteE
